synapse/tests/test_lib_lmdbslab.py

```diff
@@ -890,6 +892,40 @@ async def test_slab_mapfull_drop(self):
                 self.false(slab.dbexists('foo'))
                 self.gt(slab.mapsize, before_mapsize)
 
+    @staticmethod
+    def make_slab(path):
+        '''
+        Multiprocessing target for expanding an existing slab
+        '''
+        async def workloop():
+            s_glob.iAmLoop()
+            data = [i.to_bytes(4, 'little') for i in range(400)]
+            async with await s_lmdbslab.Slab.anit(path, map_size=32000, growsize=5000) as slab:
+                slab.initdb('foo')
+                kvpairs = [(x, x) for x in data]
+                slab.putmulti(kvpairs)
+                slab.forcecommit()
+
+        asyncio.run(workloop())
+
+    async def test_slab_mapfull_initdb(self):
+        '''
+        Test a mapfull in the middle of an initdb
+        '''
+        mpctx = multiprocessing.get_context('spawn')
+        with self.getTestDir() as dirn:
+            path = os.path.join(dirn, 'test.lmdb')
+            async with await s_lmdbslab.Slab.anit(path, map_size=32000) as slab:
+                pass
+            async with await s_lmdbslab.Slab.anit(path, map_size=32000, readonly=True) as slab:
+
+                proc = mpctx.Process(target=self.make_slab, args=(path,))
+                proc.start()
+                proc.join(10)
+                self.nn(proc.exitcode)
+                slab.initdb('foo')
+                self.true(True)
```

Comment:
is this just testing that the code doesn't go boom?

Reply:
Patch coverage of 100% seems to indicate that it covers triggering the actual resize, right?

Reply:
This tests that the previous line *didn't* hit an exception (which it did before the change).

Reply:
(It is redundant, but I know you like the explicitness)

Reply:
:+1: